cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -837,7 +843,11 @@ public MapAndStatus(T loadMap, K status) {
                 return requestRebalance(reconciliation, host, apiClient, kafkaRebalance, true, rebalanceOptionsBuilder);
             default:
                 LOGGER.warnCr(reconciliation, "Ignore annotation {}={}", ANNO_STRIMZI_IO_REBALANCE, rebalanceAnnotation);
-                return configMapOperator.getAsync(kafkaRebalance.getMetadata().getNamespace(), kafkaRebalance.getMetadata().getName()).compose(loadmap -> Future.succeededFuture(new MapAndStatus<>(loadmap, buildRebalanceStatusFromPreviousStatus(kafkaRebalance.getStatus(), validate(reconciliation, kafkaRebalance)))));
+                Set<Condition> conditions = validate(reconciliation, kafkaRebalance);
+                conditions.add(StatusUtils.buildWarningCondition("InvalidAnnotation", "Wrong annotation value: " +
```

Comment:
Why aren't you using the `validateAnnotation` method here?

Reply:
In `ProposalReady` state every annotaion is managed already in the cases if I am not wrong and the last case is when we apply some wrong annotation

Reply:
I guess I can still do it here, since the annotation will always be wrong in this case

Reply:
Added @tomncooper